src/types.js

```diff
@@ -49,6 +49,18 @@ export type InputSelectionType = {
 
 export type Account = Auth;
 
+/**
+ * A list item for all private message list
+ */
+
+export type PmConversationListItem = {
```

Comment:
The type this is really describing isn't a "list item", which is a component in the UI -- it's a summary of a PM conversation itself.

`UnreadStreamData` is a pretty decent model to make this name parallel to: so `PmConversationData`.